.gitignore

```diff
@@ -36,5 +36,6 @@ bundle-config-loader.js
 
 hooks
 .DS_Store
-
+.nyc_output
+coverage
```

Comment:
Shouldn't add these lines in `.npmignore` as well?

Reply:
Yup!